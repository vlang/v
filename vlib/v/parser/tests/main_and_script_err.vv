fn main() {
	println('main')
}

println('out')
