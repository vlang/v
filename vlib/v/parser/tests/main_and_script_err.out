vlib/v/parser/tests/main_and_script_err.vv:5:1: error: function `main` is already defined, put your script statements inside it
    3 | }
    4 | 
    5 | println('out')
      | ~~~~~~~
